platform.md

```diff
@@ -378,19 +379,21 @@ Usage:
+| `<app>`         | `CNB_APP_DIR`          | `/workspace`                                           | Path to application directory                                                                                                                                |
+| `<buildpacks>`  | `CNB_BUILDPACKS_DIR`   | `/cnb/buildpacks`                                      | Path to buildpacks directory (see [Buildpacks Directory Layout](#buildpacks-directory-layout))                                                               |
+| `<extensions>`  | `CNB_EXTENSIONS_DIR`   | `/cnb/extensions`                                      | (**[experimental](#experimental-features)**) Path to image extensions directory (see [Image Extensions Directory Layout](#image-extensions-directory-layout) |
+| `<generated>`   | `CNB_GENERATED_DIR`    | `<layers>/generated`                                   | (**[experimental](#experimental-features)**) Path to output directory for generated Dockerfiles                                                              |
+| `<group>`       | `CNB_GROUP_PATH`       | `<layers>/group.toml`                                  | Path to output group definition                                                                                                                              |
+| `<layers>`      | `CNB_LAYERS_DIR`       | `/layers`                                              | Path to layers directory                                                                                                                                     |
+| `<log-level>`   | `CNB_LOG_LEVEL`        | `info`                                                 | Log Level                                                                                                                                                    |
+| `<order>`       | `CNB_ORDER_PATH`       | `<layers>/order.toml` if present, or `/cnb/order.toml` | Path resolution for order definition (see [`order.toml`](#ordertoml-toml))                                                                                   |
+| `<plan>`        | `CNB_PLAN_PATH`        | `<layers>/plan.toml`                                   | Path to output resolved build plan                                                                                                                           |
+| `<platform>`    | `CNB_PLATFORM_DIR`     | `/platform`                                            | Path to platform directory                                                                                                                                   |
+| `<buildConfig>` | `CNB_BUILD_CONFIG_DIR` | `/cnb/config/env.build`                                | Path to operator defined environment variables                                                                                                               |
```

Comment:
```suggestion
| `<buildConfig>` | `CNB_BUILD_CONFIG_DIR` | `/cnb/build-config`                                | Path to operator-defined build config                                                                                                               |
```

Reply:
This matches the implementation in the lifecycle... also if I'm not mistaken the build config dir is the parent dir and env.build is the child dir.

Reply:
Am I looking in the wrong place when I see `build.env` in [`setupEnv`](https://github.com/buildpacks/lifecycle/blob/main/buildpack/build.go#L236)?

Reply:
I see it now as https://github.com/buildpacks/lifecycle/blob/main/platform/defaults.go#L85

Reply:
> Am I looking in the wrong place when I see `build.env` in [`setupEnv`](https://github.com/buildpacks/lifecycle/blob/main/buildpack/build.go#L236)?

You are correct!

Reply:
Ah well, now I'm confused... that function is for buildpack created env dirs. In my testing, `/cnb/build-config/env` worked.